### PR TITLE
Allow trusted ui to use visible-blurred, cautioning against text input leakage

### PR DESCRIPTION
**Pull request by Manishearth**
Fixes https://github.com/immersive-web/webxr/issues/743

Implements my proposal from https://github.com/immersive-web/webxr/issues/743#issuecomment-625969727 , pending discussion during the call.

***
<a href="https://pr-preview.s3.amazonaws.com/Manishearth/webxr/pull/1034.html" title="Last updated on May 12, 2020, 6:25 PM UTC (22af4a9)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/immersive-web/webxr/1034/fb2ff08...Manishearth:22af4a9.html" title="Last updated on May 12, 2020, 6:25 PM UTC (22af4a9)">Diff</a>

### CODE REVIEW

**Comment by toji** [comment]
Feels pretty good, curious to see what everyone else thinks on the call.

File: index.bs [hunk 1]
```diff
@@ -2384,8 +2384,9 @@ Note: Examples of [=trusted UI=] include:
...
+The ability to read input information (head pose, input pose, etc) poses a risk to the integrity of [=trusted UI=] as the page may use this information to snoop on the choices made by the user while interacting with the [=trusted UI=]. To prevent this risk the user agent MUST set the [=visibility state=] of all {{XRSession}}s to {{XRVisibilityState/"hidden"}} or {{XRVisibilityState/"visible-blurred"}} when the user is interacting with [=trusted UI=] ([=trusted immersive ui|immersive=] or non-immersive) such as URL bars or system dialogs. Additionally, to prevent a malicious page from being able to monitor input on other pages the user agent MUST set the {{XRSession}}'s [=visibility state=] to {{XRVisibilityState/"hidden"}} if the [=currently focused area=] does not belong to the document which created the {{XRSession}}.
```
Comment (cabanier) [inline index.bs:2387]: Also call out the keyboard in addition to URL bars and system dialogs

File: index.bs [hunk 2]
```diff
@@ -2384,8 +2384,9 @@ Note: Examples of [=trusted UI=] include:
...
 
+When choosing between using {{XRVisibilityState/"hidden"}} or {{XRVisibilityState/"visible-blurred"}} for a particular instance of [=trusted UI=], the user agent should consider whether head pose information is a security risk. For example, [=trusted UI=] involving text input, especially password inputs, can potentially leak the typed text through the user's head pose as they type.
```
Comment (cabanier) [inline index.bs:2389]: the user agent should consider -> MUST consider

Also, we should mention that eye tracking should be disabled

**Comment by avadacatavra** [comment]
lgtm :)

**Pull request merged**